standard-hackathon-rules.md

```diff
@@ -29,14 +29,14 @@ Remember that hackathons are like marathons. Some people go to compete but most
 
 After hacking finishes, teams will show their projects each other and to the judges.
 
-You are strongly encouraged to present a demo of what you have built. Pitches or presentations are discouraged. You are not judged on the quality of your pitch or the quality of your idea. As you are judged on what you built, you'll only hurt yourself by not showing a demo.
+You are strongly encouraged to present a demo of what you have built at the current hackathon. Pitches or presentations are discouraged. You are not judged on the quality of your pitch or the quality of your idea. As you are judged on what you built, you'll only hurt yourself by not showing a demo.
```

Comment:
This paragraph feels weirdly placed and a bit negative. What is the difference between the purpose of this paragraph and the paragraph after this one? 

If I had to choose, I'd opt for the paragraph after this, but breaking it up into the respective purposes.